### ISSUE TITLE
Pittsburgh Public Transport doesn't support PTv1

### ISSUE THREAD

**Issue by ghost** [osmandapp/OsmAnd#19311]
### Describe the routing engine used (required)

- [x] OsmAnd's in-app offline routing
- [ ] Any online routing provider (YOURS, openrouteservice, OSRM, etc.)

### Describe your routing Profile (required)

Public Transport offline, default

### Describe your start and end points (required)

4923 Centre Ave Pittsburgh PA --> PPG Paints Arena

https://www.openstreetmap.org/directions?engine=fossgis_osrm_car&route=40.4542%2C-79.9456%3B40.4396%2C-79.9895#map=13/40.4579/-79.9686

### Describe the actual route (required)

Takes 71B in Oakland, Gets off verry far from closest stop in town

### Describe the expected route (required)

Take 82 straight there from nearest stop on centre or take 71A or C and transfer to 61 or 71B in Oakland if taking 82 isnt available on time

### Describe what Maps you used (online or offline) (required)

- [ ] Offline maps offered within the OsmAnd app for download. 
- [x] Online (tile / raster) maps 

Supermaps, 
https://github.com/pnoll1/osmand_map_creation/releases

Using Pa
https://github.com/pnoll1/osmand_map_creation/releases/download/1.09/Us_pa.obf

### Your Environment (required)

```
OsmAnd Version: 4.6.12
Android/iOS version: CalyxOS 5.5.0 (Android 14)
Device model: Pixel 6
```

### Anything else? (optional)

Truetime Api is the best place for bus data in pgh if its not already added from there.

https://catalog.data.gov/dataset/true-time-api-link-real-time-arrival-info

**Comment by Arndorferd** [comment]
I dont think you understand. Its not about the nuber of changes? It should let me off much earlier then downtown on the 71B even if I walked all the way to catch it. The 2 changes makes little sense as well. I could take the 54 into Oakland and then take the 71B or the 61 buses. It instead goes all the way to the south side?

I understand that its in beta but doesnt the feedback help beta test?

**Comment by Arndorferd** [comment]
I mean the basic thing in Pittsburgh is most inbound busses lead to Oakland (where the college is) or downtown, or both. The routing system seems to not understand that the fastest route is to either go to Oakland and take a bus downtown from there or to find a bus nearby that goes directly downtown.

Yes the 71B goes directly downtown but it has you walking to oakland the 61a, 61b, 61c also go downtown from oakland. The 82 nearby goes directly downtown. The 71a, 71c, and 54 all go to oakland.

Yet it has you either walking to Oakland or taking the 54 way past Oakland. 

Both of which seem verry inefficient.

**Comment by Arndorferd** [comment]
Basically something is both wrong in its ability to understand the closest stop available on the 71B to the arena since it goes far past it even though there are closer stops. Theres also something wrong in how it judges what changes to make because it makes changes way out of the way.

**Comment by vshcherb** [comment]
We need to recheck that issue again I think, if it's faster route we need to check our settings

**Comment by DmitryAlexei** [comment]
Please note, that OsmAnd supports only Public Transport v2 ([PTv2](https://wiki.openstreetmap.org/wiki/Public_transport)) scheme tagging for navigation (also see [documentation](https://osmand.net/docs/user/navigation/routing/public-transport-navigation)). 
- This is why in Pennsylvania (and other places) OsmAnd uses only routes tagged according to PT v2 scheme. See [54C](https://www.openstreetmap.org/relation/8694237) and [71B](https://www.openstreetmap.org/relation/6547692) routes
- And this is why the fastest public transport routes from specified starting point to the end point  N [82](https://www.openstreetmap.org/relation/8703401) ,  [71A](https://www.openstreetmap.org/relation/8703207) and [71C](https://www.openstreetmap.org/relation/8703206) are not used. All these routes are tagged with PT v 1 scheme and not supported by OsmAnd.

A possible way out of this situation could be to change the tagging of public routes for version 2. However, I am not aware of the reasons why the older version is still present in Pennsylvania.
